game/scripts/vscripts/heroes/skywrath_mage/skywrath_mage_mystic_flare.lua

```diff
@@ -0,0 +1,118 @@
+
+    -- Apply a Mystic Flare on target location
+    CreateModifierThinker(caster, ability, modifier_thinker, {duration = duration}, target_point, caster:GetTeamNumber(), false)
+
+    -- Scepter: Fire a secondary Mystic Flare outside of the original flare area
+    if scepter then        
+        local scepter_target = SkywrathSpellsPositionFinder(caster, target_point, scepter_radius, radius)
+        if scepter_target then
+            CreateModifierThinker(caster, ability, modifier_thinker, {duration = duration}, scepter_target:GetAbsOrigin(), caster:GetTeamNumber(), false)
+        end         
+    end
+end
+
+
+
+-- Modifier: Thinker that is responsible for dealing damage around it
+modifier_lua_mystic_flare = class({})
+
+function modifier_lua_mystic_flare:OnCreated()
+    -- Ability properties
+    self.caster = self:GetCaster()
+    self.ability = self:GetAbility()
+    self.parent = self:GetParent()
+    self.sound_target = "Hero_SkywrathMage.MysticFlare.Target"    
+    self.core_particle = "particles/units/heroes/hero_skywrath_mage/skywrath_mage_mystic_flare_ambient.vpcf"
+    self.parent_loc = self.parent:GetAbsOrigin()
+
+    -- Ability specials
+    self.radius = self.ability:GetSpecialValueFor("radius")
+    self.damage = self.ability:GetSpecialValueFor("damage")
+    self.damage_interval = self.ability:GetSpecialValueFor("damage_interval")
+    self.duration = self.ability:GetSpecialValueFor("duration")
+
+    -- Apply particle effect
+    self.core_particle_fx = ParticleManager:CreateParticle(self.core_particle, PATTACH_WORLDORIGIN, nil)        
+    ParticleManager:SetParticleControl(self.core_particle_fx, 0 , self.parent_loc)
+    ParticleManager:SetParticleControl(self.core_particle_fx, 1, Vector(self.radius, self.duration, 0))
+    ParticleManager:ReleaseParticleIndex(self.core_particle_fx)
+
+    -- Start thinking
+    self:StartIntervalThink(self.damage_interval)
+end
+
+function modifier_lua_mystic_flare:OnIntervalThink()
+    if IsServer() then        
+        -- Play hit sound
+        EmitSoundOn(self.sound_target, self.parent)
+
+        -- Find enemies in radius
+        local enemies = FindUnitsInRadius(self.caster:GetTeamNumber(),
+                                          self.parent_loc,
+                                          nil,
+                                          self.radius,
+                                          DOTA_UNIT_TARGET_TEAM_ENEMY,
+                                          DOTA_UNIT_TARGET_HERO,
+                                          DOTA_UNIT_TARGET_FLAG_NOT_CREEP_HERO + DOTA_UNIT_TARGET_FLAG_NOT_ILLUSIONS,
+                                          FIND_ANY_ORDER,
+                                          false)
+
+        -- Calculate damage for this instance for all enemies present
+        local damage = 0 
+        if #enemies > 0 then
+            damage = (self.damage / #enemies / self.duration * self.damage_interval)
```

Comment:
`a/b/c` can be interprested as `(a/b)/c` or `a/(b/c)` which are two different things, please add parentheses here to make it so there is only one possible interpretation.